feat(Flemmix): add presence

alan7383 opened the pull request:
## Description

This pull request adds a new, comprehensive presence for the streaming website `flemmix.stream`.

The presence supports:
- Viewing a movie or a series, including episode details and timestamps.
- Browsing the homepage.
- Browsing specific categories (genres, old movies, etc.).
- Using the search functionality.
- Viewing a user profile.
- Browsing personal favorites, recent comments, and unread posts.
- Managing private messages and using the contact form.

## Acknowledgements
- [x] I read the [Activity Guidelines](https://github.com/PreMiD/Activities/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `npm run lint`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Activities/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

**Watching a series:**
<img width="1893" height="664" alt="image" src="https://github.com/user-attachments/assets/67d2d175-e45c-4e5d-8eef-ca5c467e336f" />

**Browsing a category:**
<img width="1842" height="844" alt="image" src="https://github.com/user-attachments/assets/9dc0d7ff-6991-4c0e-80f5-4a1371d4ca7e" />

**Searching:**
<img width="1919" height="888" alt="image" src="https://github.com/user-attachments/assets/0c83c7ec-1d88-4c80-898f-6df52082247a" />

</details>

theusaf commented on websites/F/Flemmix/metadata.json:
```diff
@@ -0,0 +1,53 @@
+  "url": "flemmix.stream",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/JRsbkwB.png",
+  "thumbnail": "https://i.imgur.com/JRsbkwB.png",
```
The thumbnail should be a landscape promotional image or screenshot

alan7383 commented:
ok thx, removed the declare stuff from both files, should use globals now. also fixed my bad on the deprecated `getTimestamps`, using the proper import now.
lmk if it’s all good!

Artoria01 commented:
@alan7383 I’ve pulled your branch with the new links and added myself as a contributor. Please check your branch and pull if everything looks good. I can also take over updating the next links if needed.

[Pull Request](https://github.com/alan7383/Activities/pull/1)
